### PR TITLE
Allows the Dashboard importer to be built through the makefiles 

### LINKED ISSUE #1110: Dashboard importer is pulled from Dockerhub instead of being built
### Describe the bug

When running the persistence elastic hook, The Dashboard importer is required. The image for it is pulled from Dockerhub instead of being built through our makefiles. This results in having the wrong version or an inexistent version getting used (see #1095 ).

### Steps To Reproduce
cd ./secureCodeBox/hooks/persistence-elastic
make docker-build
make docker-export
make kind-import
rm *.tar  # clean up
make deploy
make deploy-test-deps
make integration-tests

### Expected behavior

As is the behaviour of our scanners, the image for the dashboard importer should be built within the CI pipeline and tagged with the current git SHA tag.

### System (please complete the following information):
 - secureCodeBox v3.9.1
 - OS: Linux 5.16.15-76051615-generic
 - Kubernetes Version: v1.23.5
 - Docker Version: Docker version 20.10.14, build a224086

### Additional context

This issue was found as a result of #1095

### PR DESCRIPTION
**Pull request by Ilyesbdlala**
## Description

closes #1110

### CODE REVIEW

**Review by Weltraumschaf — DISMISSED** [review]

File: hooks/persistence-elastic/Makefile [hunk 1]
```diff
@@ -37,7 +37,7 @@ dashboard-importer-kind-import:
 
 deploy: ## 💾 Deploy this module via HelmChart into namespace "integration-tests"
 	@echo ".: 💾 Deploying '$(name)' $(hook-prefix) HelmChart with the docker tag '$(IMG_TAG)' into kind namespace 'integration-tests'."
-	helm -n integration-tests upgrade --install $(name) . --timeout 10m --wait  \
+	helm -n integration-tests upgrade --install $(name) . --timeout 20m --wait  \
```
Comment (Weltraumschaf) [inline hooks/persistence-elastic/Makefile:40]: 😂

**Review by Weltraumschaf — DISMISSED** [review]

File: hooks/persistence-elastic/Makefile (hunk 1 above)
Comment (Ilyesbdlala) [reply hooks/persistence-elastic/Makefile:40]: The problem right now is that the deployment of persistence-elastic locally takes at least 18min. I set the timeout 20min to see if that's enough. But it doesn't seem like it. (so expect another similar commit :joy:) This probably has to be moved to a nightly build.

Comment (J12934) [reply hooks/persistence-elastic/Makefile:40]: @Ilyesbdlala do you know why this is taking so long?

```bash
helm upgrade --install persistence-elastic secureCodeBox/persistence-elastic --wait
```

Takes 3 min on my local machine, and 2 min on the second run (once the images were cached)

Comment (Ilyesbdlala) [reply hooks/persistence-elastic/Makefile:40]: @J12934  That was the case for me until I deleted my kind cluster and created a new one to have the same initial state as the CI. The first run takes about 18 min, the next one is less than 1 minute ( after a `helm delete persistence-elastic`)
The problem is reproducible through the following:
```bash
kind delete cluster
kind create cluster

# install SCB
helm repo add secureCodeBox https://charts.securecodebox.io
kubectl create namespace securecodebox-system
helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator

# install persistence-elastic
cd hooks/persistence-elastic
make docker-build
make docker-export
make kind-import
make deploy # this is where the issue is.
```

Comment (SebieF) [reply hooks/persistence-elastic/Makefile:40]: > The problem is reproducible through the following:
> 
> ```shell
> kind delete cluster
> kind create cluster
> 
> # install SCB
> helm repo add secureCodeBox https://charts.securecodebox.io
> kubectl create namespace securecodebox-system
> helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator
> 
> # install persistence-elastic
> cd hooks/persistence-elastic
> make docker-build
> make docker-export
> make kind-import
> make deploy # this is where the issue is.
> ```

Comment (Ilyesbdlala) [reply hooks/persistence-elastic/Makefile:40]: > > The problem is reproducible through the following:
> > ```shell
> > kind delete cluster
> > kind create cluster
> > 
> > # install SCB
> > helm repo add secureCodeBox https://charts.securecodebox.io
> > kubectl create namespace securecodebox-system
> > helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator
> > 
> > # install persistence-elastic
> > cd hooks/persistence-elastic
> > make docker-build
> > make docker-export
> > make kind-import
> > make deploy # this is where the issue is.
> > ```
> 
> For me, it took only about 3-5 min for make deploy to run locally (both on the main branch and the PR branch).. I had to execute `kubectl create namespace integration-tests` and `cd hook-sdk/nodjs` `make docker-build` in addition to the steps above. I checked by executing `kubectl -n integration-test get pods` (persistence-elastic and elasticsearch-master are in state _Running_). OS: Ubuntu 20.04

Yeah, it seems my theory, that the error is due to the amount of time required for persistence elastic to deploy, is false.
The error log from the CI is not helpful. I'm trying to figure out if there's a flag I can set to allow for more debug logs.

Comment (J12934) [reply hooks/persistence-elastic/Makefile]: The pods have been pending over the entire time, meaning that something is missing for kubernetes to start the pods (see the Inspect Post Failure step: https://github.com/secureCodeBox/secureCodeBox/runs/6286731595?check_suite_focus=true#step:27:31)

I've added a `kubectl describe pods` over the integration-tests namespace. This prints out the detailed output for the pods, which should tell us the reason why kubernetes can't start the pods.

Comment (J12934) [reply hooks/persistence-elastic/Makefile]: The hosted github action runners are apperently too small to run elasticsearch...
https://github.com/secureCodeBox/secureCodeBox/runs/6294995302?check_suite_focus=true#step:27:228

**Comment by Ilyesbdlala** [comment]
This PR closes the issue #1110. A new issue #1165 was discovered.

**Review by J12934 — APPROVED** [review]

**Pull request merged**